### PR TITLE
Update smoke tests

### PR DESCRIPTION
**Pull request by mdr0id**
Removing Sprout funding logic for upcoming NU. Also removes need to have active Sprout TAZ wallet to manage, to make these tests valid. I preserved the test numbers for a more dynamic approach later. Now they can serve as historical reference for folks wondering about Sprout coverage.

### CODE REVIEW

File: qa/zcash/smoke_tests.py [hunk 1]
```diff
@@ -166,6 +166,7 @@ def wait_for_balance(zcash, zaddr, expected=None):
                     ttl = 300
 
 def wait_and_check_balance(results, case, zcash, addr, expected):
+    time.sleep(5)
```
Comment (daira) [inline qa/zcash/smoke_tests.py:169]: Why is this needed?

Comment (daira) [inline qa/zcash/smoke_tests.py:169]: Add a comment explaining why this is needed.

**Review by daira — DISMISSED** [review]
utACK

**Comment by mdr0id** [comment]
Do not merge yet. I'm looking through the squash flows and using this to sanity check the wonky UI flows that clutter git logs

**Comment by mdr0id** [comment]
Closing this in favor of https://github.com/zcash/zcash/pull/5293
